Simple logger using Express middleware

smartclash opened the pull request.

Commit:
Merge pull request #1 from brijeshshah13/master

Update fork

Commit:
New simple logger that uses middleware

brijeshshah13 reviewed (commented):
Please attach the README.md file with necessary instructions. You can view a similar file from the other apps.

Commit:
Gitignore used

Commit:
New awesome changes

smartclash commented:
Hey @brijeshshah13,

Now, in this PR, I've added more features. Removed `node_modules` folder as it looks noobical. And, I've started to sign the commits because I am afraid of someone getting access to my commit credentials. So, we are secure - in enterprise level :P 

![GPG signing now up](https://user-images.githubusercontent.com/16922883/33318495-d0f95152-d460-11e7-81c4-2ff6d3f8d04f.png)

brijeshshah13 commented:
Appreciate your work @smartclash. The commits from this PR have been pushed to `master`. Following this update, I'm now closing this PR.
